### PR TITLE
Add ability to specify event provider to event description

### PR DESCRIPTION
**Pull request by wpf500**
Specify an event provider with the description `<!-- provider: foo -->`. This will attempt to use a logo called `.icon-foo` instead of the standard icon

image

### CODE REVIEW

File: frontend/test/resources/model/eventbrite/event-with-provider.json [hunk 1]
```diff
@@ -0,0 +1,119 @@
+{
+    "resource_uri": "https://www.eventbriteapi.com/v3/events/14506809257/",
+    "name": {
+        "text": "Essentials of pigeon grooming with Will Franklin",
+        "html": "Essentials of pigeon grooming with Will Franklin"
+    },
+    "description": {
+        "text": "\nWill Franklin, famed pigeon-fancier, reveals at long last to the world his time-honoured techniques for grooming his flock of racing pigeons. Hear how the bespectacled bird-beautifier makes household names like Bob the Pigeon into a sleek, well-coiffured avian eye-catcher, and learn how to use just the right amount of Morton's Bird Shampoo to keep your pigeon's feathers looking lustrous. Will plans to close the session with a live demonstration of his infamous \"Pigeon Orchestra\", where he accompanies himself on ukulele, backed by a cooing choir of pigeons, performing his own compositions from his concept album \"Dark Side of the Pigeon\". A rare treat. ",
```
Comment (mattandrews) [inline frontend/test/resources/model/eventbrite/event-with-provider.json]: I love that this is now permanently in the repo (and public)

**Comment by mattandrews** [comment]
:+1:

**Comment by davidrapson** [comment]
Actually, just had a chat with Will about trying to separate our UI icons from content imagery like sponsors. So going to change the way that image gets loaded slightly… so :-1:  for now.

**Comment by feedmypixel** [comment]
I have had a look into how the image is handled responsively - BenW is not currently happy with it or with making the logo just drop down underneath and is coming up with a solution for this.

**Comment by feedmypixel** [comment]
> I have had a look into how the image is handled responsively - BenW is not currently happy with it or with making the logo just drop down underneath and is coming up with a solution for this.

Agreed - lets do this as a separate pull request.

**Comment by wpf500** [comment]
@jamesoram

**Comment by jennysivapalan** [comment]
:+1: :bird:

**Comment by jamesoram** [comment]
:+1:

**Pull request merged**